python/orca/dev/test/run-pytests-spark-openvino.sh

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+cd "`dirname $0`"
+
+export PYSPARK_PYTHON=python
+export PYSPARK_DRIVER_PYTHON=python
+
+ray stop -f
+
+cd ../../
+
+echo "Running openvino test"
+python -m pytest -v test/bigdl/orca/learn/spark/test_estimator_openvino.py::TestEstimatorForOpenVINO::test_openvino_predict_ndarray
+python -m pytest -v test/bigdl/orca/learn/spark/test_estimator_openvino.py::TestEstimatorForOpenVINO::test_openvino_predict_ndarray_multi_input
+python -m pytest -v test/bigdl/orca/learn/spark/test_estimator_openvino.py::TestEstimatorForOpenVINO::test_openvino_predict_xshards
+# uncomment below pytests to run corresponding testing function
+# python -m pytest -v test/bigdl/orca/learn/spark/test_estimator_openvino.py::TestEstimatorForOpenVINO::test_openvino_multi_output
+# python -m pytest -v test/bigdl/orca/learn/spark/test_estimator_openvino.py::TestEstimatorForOpenVINO::test_openvino_predict_spark_df
```

Comment:
add some comments why these two tests can't run?

Reply:
@cyita ?

Reply:
Will enable these two tests in another pr today.